ui/components/multichain/multi-srp/import-srp/import-srp.tsx

```diff
@@ -230,132 +233,163 @@ export const ImportSrp = ({
   );
 
   return (
-    <Box
-      display={Display.Flex}
-      flexDirection={FlexDirection.Column}
-      height={BlockSize.Max}
-      data-testid="import-srp-container"
-    >
-      <Text variant={TextVariant.bodyMd} marginTop={2}>
-        {t('importSRPDescription')}
-      </Text>
+    <Page className="import-srp__multi-srp">
+      <Header
+        startAccessory={
+          <ButtonIcon
+            ariaLabel="back"
+            iconName={IconName.ArrowLeft}
+            onClick={() => {
+              history.push(DEFAULT_ROUTE);
+            }}
+          />
+        }
+        endAccessory={
+          <ButtonIcon
+            ariaLabel="close"
+            iconName={IconName.Close}
+            onClick={() => {
+              history.push(DEFAULT_ROUTE);
+            }}
+          />
+        }
+      >
+        {'Import Secret Recovery Phrase'}
```

Comment:
We should probably have a `importSRPPage` message for this one?

Reply:
Yeah, this should be localized.